modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,102 @@
+
+image::organization-2b25c.png[]
+
+Since business groups are hierarchical, the owner of a parent business group always retains administrator permissions for any child business group within that parent. This means that the owner of a business group can create child business groups and assign another user as the owner (owners automatically have the *Organization Administrators* role) of those child business groups. The owner of the parent group remains an *Organization Administrator* within the child business group. 
+
+Conversely, owners of child business groups cannot:
+
+* Access or modify the parent business group or master organization settings
+* View the parent business group's client ID and client secret
+
+When you create a child business group within a parent, only the vCores that were assigned to the parent business group can be allocated to the child.
+
+=== Navigating Between Business Groups
+
+When your organization has multiple business groups, you can navigate between them using the menu at the top-right corner. Switching between business groups changes the list of available CloudHub deployments, APIs, users, and roles settings.
+
+image::switch-suborg.png[switch+suborg]
+
+[NOTE]
+--
+When you sign into Anypoint Platform, you are signed back into the organization or business group you were in when you last signed out.
+--
+
+If you are not an organization administrator, you can view only the business groups that you have permissions to view. In the Organization tab, the tree of your organization displays only the business groups you are a member of.
+
+[[creatingroles]]
+=== Creating Roles and Handling Membership to Business Groups
+
+Roles can be applied at the master organization level as well as at the business group level. However, each controls different resources. Roles and permissions assigned to a user in one particular business group (or the master org) are independent of the roles and permissions assigned to that same user in other business groups.
+
+To obtain the membership for a business group, you must be granted a role within that business group. Once you are visible in a business group, you can be granted individual permissions in that business group. All users, regardless of which business groups they are in, belong to the master organization and can be assigned individual permissions in the master organization.
+
+Members that are added to a business group can view and access the business group, but to access certain resources, you must have the appropriate roles assigned to you. For example, to access APIs and CloudHub deployments in the business group, you must be assigned roles that belong to that business group. Additionally, roles that belong to a business group can only grant access to APIs and CloudHub deployments within that same business group.
```

Comment:
how are APIs or deployments (!) associated with a business group? "in" is confusing me (just me?)

Reply:
you only need to tell me this in one place (first section), and if it's there and I didn't get it, my apologies. I understand how users & roles are associated with a biz group, but not other objects like APIs or *deployments*?! Is the runtime engine associated? apps in the runtime engine?